db/version_set.cc

```diff
@@ -1986,12 +1988,14 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
     tracing_get_id = vset_->block_cache_tracer_->NextGetId();
   }
 
+  ReadOptions read_opts = read_options;
+  read_opts.rate_limiter_priority = Env::IO_USER;
```

Comment:
I don't think we need to do this here, since the user is already passing read_options, which should have their desired priority.

Reply:
This was changed in a following commit.